health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/fetchFromMicroplan.js

```
@@ -0,0 +1,197 @@
+        planConfigurationId
+      );
+
+      setMicroplan({
+        ...fetchFromMicroplanResponse,
+      });
+    }
+  }, [templates]);
+  useEffect(() => {
+    const fetchMicroplanData = async () => {
+      try {
+        const { id, tenantId } = data?.updatedCampaignData;
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)